### ISSUE TITLE
JaxRS application shown as component type "JSP" on admin console

### ISSUE THREAD

**Issue by ggam** [payara/Payara#1808]
# Description #
----------

Admin console shows component type "JSP" for JAX-RS Application subclasses on the deployment details page.

## Expected Outcome

I expect it to be a JAX-RS component.

## Current Outcome

![imagen](https://user-images.githubusercontent.com/2109040/28668228-2a92906e-72cf-11e7-94ba-994034d9ae7a.png)

## Steps to reproduce (Only for bug reports) 

Deploy a WAR with a class like the following:
```
package com.example.test;

import javax.ws.rs.ApplicationPath;
import javax.ws.rs.core.Application;

@ApplicationPath("resources")
public class RestConfig extends Application {

}
```

Then go to the admin console and see the deployment details.

## Environment ##

- **Payara Version**: 4.1.2.173-nightly

**Comment by fturizo** [comment]
Created internal JIRA Issue `PAYARA-1906`

**Comment by smillidge** [comment]
Fixed by #1978
